### ISSUE TITLE
Support for  Material Icons 

### ISSUE THREAD

**Issue by TomerconDevelopers** [DavBfr/dart_pdf#383]
Material design Icon is a great feature in Flutter. It would be nice if we can add these in PDF too

**Comment by SmiffyKMc** [comment]
Has this progressed since? Would be great to have.

**Comment by DavBfr** [comment]
Implemented in pdf-1.13.0.

example:
```dart
import 'dart:io';

import 'package:pdf/pdf.dart';
import 'package:pdf/widgets.dart' as pw;

void main() {
  final doc = pw.Document();
  final data = File('material.ttf').readAsBytesSync();
  final material = pw.Font.ttf(data.buffer.asByteData());

  doc.addPage(
    pw.Page(
      build: (pw.Context context) => pw.Center(
        child: pw.Icon(IconData(0xe000, font: material, size: 50, color: PdfColors.blueGrey),
      ),
    ),
  );

  final file = File('example.pdf');
  file.writeAsBytesSync(doc.save());
}
```

The IconData constants are not predefined so any icon font file can be used, not limited to Material.
Don't hesitate to [Buy Me A Coffee](https://www.buymeacoffee.com/JORBmbw9h).
